dss/operations/storage.py

```diff
@@ -108,6 +108,26 @@ def process_key(self, key):
                                       file_metadata_content_type=file_metadata['content-type'],
                                       blob_content_type=blob_content_type))
 
+@storage.action("repair-file-blob-metadata",
+                arguments={"--entity-type": dict(default="file", choices=["file"])})
```

Comment:
What other choices other than file will we need to account for in the future?

Reply:
None, this line is there to restrict the options of `--entity-type` to just `file`.